src/core/funcbloc.h

```diff
@@ -298,9 +298,7 @@ class CFunctionBlock : public forte::core::CFBContainer {
     void receiveInputEvent(TEventID paEIID, CEventChainExecutionThread *paExecEnv) {
       FORTE_TRACE("InputEvent: Function Block (%s) got event: %d (maxid: %d)\n", CStringDictionary::get(getInstanceNameId()), paEIID, getFBInterfaceSpec().mNumEIs - 1);
 
-      #ifdef FORTE_TRACE_CTF
-        traceInputEvent(paEIID);
-      #endif
+
```

Comment:
Moving this down changes the order of generated traces breaking trace analysis of existing users. Can you please restore that again.

Reply:
I reverted it back